src/baldr/graphtile.cc

```diff
@@ -257,27 +257,24 @@ void GraphTile::AssociateOneStopIds(const GraphId& graphid) {
 
 std::string GraphTile::FileSuffix(const GraphId& graphid) {
   /*
-  if you have a graphid where level == 8 and tileid == 24134109851
-  you should get: 8/024/134/109/851.gph
-  since the number of levels is likely to be very small this limits
-  the total number of objects in any one directory to 1000, which is an
-  empirically derived good choice for mechanical harddrives
-  this should be fine for s3 (even though it breaks the rule of most
-  unique part of filename first) because there will be just so few
+  if you have a graphid where level == 8 and tileid == 24134109851 you should get: 8/024/134/109/851.gph
+  since the number of levels is likely to be very small this limits the total number of objects in any one directory to 1000
+  which is an empirically derived good choice for mechanical hard drives this should be fine for s3 as well
+  (even though it breaks the rule of most unique part of filename first) because there will be just so few
   objects in general in practice
   */
 
   //figure the largest id for this level
-  auto level = TileHierarchy::levels().find(graphid.level());
-  if(level == TileHierarchy::levels().end() &&
-     graphid.level() == ((TileHierarchy::levels().rbegin())->second.level + 1))
-    level = TileHierarchy::levels().begin();
```

Comment:
this was very bad. it used level `0` for level `3`. which made directories like `3/000/000.gph` however level `3` has a max of id `1036799` so any transit tile happening in the extreme north would be written in the extreme south